### PR TITLE
Update to Unity 2017.41f1

### PR DESCRIPTION
**Pull request by KiwifruitDev**
## Status

- [ ] **Ready**
- [ ] **In development** 
- [x] **Hold** 

## Description

Opened the project in Unity Hub 2017.41f1, an LTS build. The results have been committed.

## Testing

Open the project in Unity 2017.41f1 (May require a Unity Hub version). You shouldn't encounter any prompt asking you to reimport.
## Concerns, notes, etc.

May result in instability somewhere, haven't done much testing.

### CODE REVIEW

**Comment by TeamPopplio** [comment]
Oops, I accidentally left the project as a Unity Hub project. Might not be an issue but I'm not too sure.

**Comment by TeamPopplio** [comment]
Just pulled latest changes, going to open it in Unity and upload any results.

File: Pokemon Unity/Pokemon Unity.sln [hunk 1]
```diff
@@ -1,19 +1,11 @@
...
+Microsoft Visual Studio Solution File, Format Version 11.00
+# Visual Studio 2010
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Pokemon Unity", "Assembly-CSharp.csproj", "{10F42E31-4957-FD75-9899-6B6E9B1C9EA8}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Assembly-CSharp-firstpass", "Assembly-CSharp-firstpass.csproj", "{689E422C-B77F-5B31-E119-AD21E63C6EE7}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Pokemon Unity", "Assembly-CSharp-firstpass.csproj", "{689E422C-B77F-5B31-E119-AD21E63C6EE7}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Assembly-CSharp-Editor", "Assembly-CSharp-Editor.csproj", "{F6278824-EF26-0B19-80EC-89090458C325}"
-EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "UnitTestProject", "..\UnitTestProject\UnitTestProject.csproj", "{7C20DCE1-E6DA-4E84-9CEC-93626D43CDFD}"
```
Comment (herbertmilhomme) [inline Pokemon Unity/Pokemon Unity.sln]: Everything looks good, except for this part here. Where unity deletes the unit testing project from Solution. This file here can be reverted/ignored (doesnt need to be changed/updated, and wont mess with importing meta data all over again).

Comment (TeamPopplio) [reply Pokemon Unity/Pokemon Unity.sln]: Er, I don't get what that means? Do you want me to revert it back to how it is in beta?

Comment (herbertmilhomme) [reply Pokemon Unity/Pokemon Unity.sln]: just this one file, not the whole commit. If you cant do it, i can do it

Comment (TeamPopplio) [reply Pokemon Unity/Pokemon Unity.sln]: Yeah that'll be fine, you can merge the changes afterwards. Thanks for letting me upgrade.

**Pull request merged**